eurocropsml/dataset/download.py

```diff
@@ -47,23 +47,19 @@ def _download_file(
         logger.info(f"{local_path} will not be downloaded again.")
 
 
-def get_user_choice() -> list[str]:
+def get_user_choice(files_to_download: list[str]) -> list[str]:
     """Get user choice for which files to download."""
-    choice = input(
-        "Would you like to download Sentinel-1 and/or Sentinel-2 data? Please enter "
-        " 'S1', 'S2', or 'both': "
-    )
-    if choice not in {"S1", "S2", "both"}:
+    print("Choose one or more of the following options by typing their numbers (e.g., 1 3):")
+    for i, file in enumerate(files_to_download, 1):
+        print(f"{i}. {file}")
+    choice = input("Enter your choices separated by spaces: ")
```

Comment:
Done

Reply:
I was thinking for consistency to also use `typer.prompt(...)`instead of `input(...)` like we use `typer.confirm(...)` in other places throughout here. Or does something speak against it?

Reply:
Ahh I see. You're right, sorry.